<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlotLabelPropertiesUI</class>
 <widget class="QDialog" name="PlotLabelPropertiesUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>341</width>
    <height>505</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Modify Label Properties</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/res/ball.ico</normaloff>:/res/ball.ico</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>X Label</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Family</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" colspan="2">
       <widget class="QComboBox" name="cbFont"/>
      </item>
      <item row="0" column="3">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Weight</string>
        </property>
       </widget>
      </item>
      <item row="0" column="4" colspan="3">
       <widget class="QComboBox" name="cbWeight"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Color</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="cbColor"/>
      </item>
      <item row="1" column="2" colspan="3">
       <widget class="QPushButton" name="cmdCustom">
        <property name="text">
         <string>Custom..</string>
        </property>
       </widget>
      </item>
      <item row="1" column="5">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Size</string>
        </property>
       </widget>
      </item>
      <item row="1" column="6">
       <widget class="QSpinBox" name="cbSize">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>72</number>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="7">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Legend Label</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="txtLegend"/>
        </item>
       </layout>
      </item>
      <item row="3" column="0" colspan="4">
       <widget class="QCheckBox" name="chkTicks">
        <property name="text">
         <string>Apply style to X tick labels</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Y Label</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Family</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" colspan="2">
       <widget class="QComboBox" name="cbFont_y"/>
      </item>
      <item row="0" column="3">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Weight</string>
        </property>
       </widget>
      </item>
      <item row="0" column="4" colspan="3">
       <widget class="QComboBox" name="cbWeight_y"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>Color</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="cbColor_y"/>
      </item>
      <item row="1" column="2" colspan="3">
       <widget class="QPushButton" name="cmdCustom_y">
        <property name="text">
         <string>Custom..</string>
        </property>
       </widget>
      </item>
      <item row="1" column="5">
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>Size</string>
        </property>
       </widget>
      </item>
      <item row="1" column="6">
       <widget class="QSpinBox" name="cbSize_y">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>72</number>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="7">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_10">
          <property name="text">
           <string>Legend Label</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="txtLegend_y"/>
        </item>
       </layout>
      </item>
      <item row="3" column="0" colspan="4">
       <widget class="QCheckBox" name="chkTicks_y">
        <property name="text">
         <string>Apply style to Y tick labels</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>120</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>cbFont</tabstop>
  <tabstop>cbWeight</tabstop>
  <tabstop>cbColor</tabstop>
  <tabstop>cmdCustom</tabstop>
  <tabstop>cbSize</tabstop>
  <tabstop>txtLegend</tabstop>
  <tabstop>chkTicks</tabstop>
  <tabstop>cbFont_y</tabstop>
  <tabstop>cbWeight_y</tabstop>
  <tabstop>cbColor_y</tabstop>
  <tabstop>cmdCustom_y</tabstop>
  <tabstop>cbSize_y</tabstop>
  <tabstop>txtLegend_y</tabstop>
  <tabstop>chkTicks_y</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>PlotLabelPropertiesUI</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>PlotLabelPropertiesUI</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
